packages/controller/src/system/rules.ts

```diff
@@ -274,838 +274,510 @@ export default () => [
+        expr: 'ceil(\n  sum by (cluster, namespace) (cortex_overrides{limit_name="ingestion_rate"})\n  * 0.59999999999999998 / 80000\n)\n',
+        labels: {
+          deployment: "ingester",
+          reason: "sample_rate_limits"
+        },
+        record: "cluster_namespace_deployment_reason:required_replicas:count"
       },
       {
-        expr: '1 -\n(\n  sum by (namespace, job) (rate(cortex_gw_request_duration_seconds_bucket{status_code!~"error|5..",le="2.5",route="cortex-read"}[5m]))\n/\n  sum by (namespace, job) (rate(cortex_gw_request_duration_seconds_count{route="cortex-read"}[5m]))\n)\n',
-        record:
-          "namespace_job:cortex_gw_read_slo_errors_per_request:ratio_rate5m"
+        expr: 'ceil(\n  (sum by (cluster, namespace) (\n    cortex_ingester_tsdb_storage_blocks_bytes{job=~".+/ingester.*"}\n  ) / 4)\n    /\n  avg by (cluster, namespace) (\n    memcached_limit_bytes{job=~".+/memcached"}\n  )\n)\n',
+        labels: {
+          deployment: "memcached",
+          reason: "active_series"
+        },
+        record: "cluster_namespace_deployment_reason:required_replicas:count"
       },
       {
-        expr: '1 -\n(\n  sum by (namespace, job) (rate(cortex_gw_request_duration_seconds_bucket{status_code!~"error|5..",le="2.5",route="cortex-read"}[30m]))\n/\n  sum by (namespace, job) (rate(cortex_gw_request_duration_seconds_count{route="cortex-read"}[30m]))\n)\n',
+        expr: 'sum by (cluster, namespace, deployment) (\n  label_replace(\n    label_replace(\n      node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate,\n      "deployment", "$1", "pod", "(.*)-(?:([0-9]+)|([a-z0-9]+)-([a-z0-9]+))"\n    ),\n    \n    \n    "deployment", "$1", "deployment", "(.*?)(?:-zone-[a-z])?"\n  )\n)\n',
         record:
-          "namespace_job:cortex_gw_read_slo_errors_per_request:ratio_rate30m"
+          "cluster_namespace_deployment:container_cpu_usage_seconds_total:sum_rate"
       },
       {
-        expr: '1 -\n(\n  sum by (namespace, job) (rate(cortex_gw_request_duration_seconds_bucket{status_code!~"error|5..",le="2.5",route="cortex-read"}[1h]))\n/\n  sum by (namespace, job) (rate(cortex_gw_request_duration_seconds_count{route="cortex-read"}[1h]))\n)\n',
+        expr: '(cluster, namespace, deployment) (\n    label_replace(\n      label_replace(\n        kube_pod_container_resource_requests_cpu_cores,\n        "deployment", "$1", "pod", "(.*)-(?:([0-9]+)|([a-z0-9]+)-([a-z0-9]+))"\n      ),\n      \n      \n      "deployment", "$1", "deployment", "(.*?)(?:-zone-[a-z])?"\n    )\n  )\n)\nor\n\n\n(\n  sum by (cluster, namespace, deployment) (\n    label_replace(\n      label_replace(\n        kube_pod_container_resource_requests{resource="cpu"},\n        "deployment", "$1", "pod", "(.*)-(?:([0-9]+)|([a-z0-9]+)-([a-z0-9]+))"\n      ),\n      \n      \n      "deployment", "$1", "deployment", "(.*?)(?:-zone-[a-z])?"\n    )\n  )\n)\n',
```

Comment:
```suggestion
        expr: '(sum by (cluster, namespace, deployment) (\n    label_replace(\n      label_replace(\n        kube_pod_container_resource_requests_cpu_cores,\n        "deployment", "$1", "pod", "(.*)-(?:([0-9]+)|([a-z0-9]+)-([a-z0-9]+))"\n      ),\n      \n      \n      "deployment", "$1", "deployment", "(.*?)(?:-zone-[a-z])?"\n    )\n  )\n)\nor\n\n\n(\n  sum by (cluster, namespace, deployment) (\n    label_replace(\n      label_replace(\n        kube_pod_container_resource_requests{resource="cpu"},\n        "deployment", "$1", "pod", "(.*)-(?:([0-9]+)|([a-z0-9]+)-([a-z0-9]+))"\n      ),\n      \n      \n      "deployment", "$1", "deployment", "(.*?)(?:-zone-[a-z])?"\n    )\n  )\n)\n',
```

Reply:
Pushed this fix.

I'm not sure how this happened.  I've checked from the [source](https://github.com/grafana/cortex-jsonnet/blob/dccf32abaa4af443d2b724edc80a7a28ec504c24/cortex-mixin/recording_rules.libsonnet#L229) through my process and can only conclude it was human error in the last possible stage.